File: examples/src/Text/DefaultFonts.cs
using System.IO

import aspose.slides as slides
using System.Drawing.Imaging
using Aspose.slides.Export

namespace Aspose.slides.Examples.CSharp.text
{
    public class DefaultFonts
    {
        public static void Run()
        {
            #ExStart:DefaultFonts
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Text()

            # Use load options to define the default regualr and asian fonts# Use load options to define the default regualr and asian fonts
            loadOptions = slides.LoadOptions(LoadFormat.Auto)
            loadOptions.DefaultRegularFont = "Wingdings"
            loadOptions.DefaultAsianFont = "Wingdings"

            # Load the presentation
            using (Presentation pptx = new Presentation(dataDir + "DefaultFonts.pptx", loadOptions))
            {
                # Generate slide thumbnail
                pptx.slides[0].GetThumbnail(1, 1).save(dataDir + "output_out.png", ImageFormat.Png)

                # Generate PDF
                pptx.save(dataDir + "output_out.pdf", SaveFormat.Pdf)

                # Generate XPS
                pptx.save(dataDir + "output_out.xps", SaveFormat.Xps)
            }
            #ExEnd:DefaultFonts
        }
    }
}

File: examples/src/Text/DefaultFonts.py
import aspose.slides as slides
import aspose.pydrawing as drawing


#ExStart:DefaultFonts
# The path to the documents directory.
dataDir = "./examples/data/"
outDir = "./examples/out/"

# Use load options to define the default regualr and asian fonts# Use load options to define the default regualr and asian fonts
loadOptions = slides.LoadOptions(slides.LoadFormat.AUTO)
loadOptions.default_regular_font = "Wingdings"
loadOptions.default_asian_font = "Wingdings"

# Load the presentation
with slides.Presentation(dataDir + "text_default_fonts.pptx", loadOptions) as pptx:
    # Generate slide thumbnail
    pptx.slides[0].get_thumbnail(1, 1).save(outDir + "text_default_fonts_out.png", drawing.imaging.ImageFormat.png)

    # Generate PDF
    pptx.save(outDir + "text_default_fonts_out.pdf", slides.export.SaveFormat.PDF)

    # Generate XPS
    pptx.save(outDir + "text_default_fonts_out.xps", slides.export.SaveFormat.XPS)
#ExEnd:DefaultFonts